File: aula07.py
#aula 07 - Operadores Aritméticos

#Renan Souza - 08/03/2021
#Entrada de dados
n1 = int(input('Digite um número: '))
n2 = int(input('Digite outro número: '))

#Manipulação de dados
s = n1 + n2
m = n1 * n2
d = n1 / n2
di = n1 // n2 #Serve para divisão inteira
e = n1 ** n2 #Serve para potências

#Saída dos dados
print('Soma: {}, Produto: {} e a Divisão: {:.1f}'.format(s, m, d), end=' ') #:.1f -> Serve para identificar as casas decimais
print('Divisão inteira: {} e Exponênciação: {}'.format(di, e))

#Fim

File: 07/Aula07.java
import java.util.Scanner;

public class Aula07 {
    public static void main(String[] args) {
        int n1;
        int n2;

        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite um número: ");
        n1 = scanner.nextInt();

        System.out.print("Digite outro número: ");
        n2 = scanner.nextInt();

        scanner.close();

        int s = n1 + n2;
        int m = n1 * n2;
        float d = n1 / n2;
        int di = n1 / n2;
        double e = Math.pow(n1, n2);

        System.out.printf("Soma: %d, Produto: %d e a Divisão: %.1f \n", s, m, d);
        System.out.printf("Divisão inteira: %d e Exponênciação: %.0f \n", di, e);
    }
}
